src/recensio/plone/browser/review.py

```diff
@@ -499,6 +504,10 @@ def show_dara_update(self):
         except AttributeError:
             return False
 
+    def get_citation_string(self):
+        # XXX specific to subtypes; implement in specific views
+        return "XXX"
```

Comment:
```suggestion
        raise NotImplementedError("Specific to subtypes; implement in a specific view")
```

Reply:
Won't that break the view for all content types except the one that currently implements this method?

Reply:
Given your comment I got that this was going to be used only for specific types...
Do we have a default?
If yes change the comment to `Implement a default`, if no I think my suggestion is valid

Reply:
No, there needs to be a specific implementation per content type. In the long run I would do it like in your suggestion, but right now (IIUC) that would prevent calling the view on all review types except one, which I think would hinder development.